lib/SauceLabs.js

```diff
@@ -234,13 +237,10 @@ SauceLabs.prototype.send = function (message, callback) {
   // Make the request.
   var options = extend({}, this.options, {
     method: method,
-    path:   path,
-
-    headers: {
-      'Accept':         'application/json',
-      'Content-Type':   'application/json',
-      'Content-Length': body != null ? body.length : 0
-    }
+    uri:   'https://saucelabs.com' + path,
```

Comment:
while we're here, let's change this to `url.format(urlObj)` - constructing urls is never a good idea this way